test/globalize_validations_test.rb

```diff
@@ -92,9 +103,9 @@ class PageWithComposedLocale < ActiveRecord::Base
 
     assert page.invalid?, "Can't be valid with title present for all locales"
 
-    assert_equal ["can't be blank"],  page.errors[:title_en]
-    assert_equal ["can't be blank"],  page.errors[:title_fr]
-    assert_equal ["can't be blank"],  page.errors[:title_es]
+    assert_equal ["can't be blank"],  (page.errors[:title_en].map { |e| e.tr("’", "'") })
```

Comment:
@pankopiotr why do we now need `e.tr("’", "'") ? 🤔

Reply:
We run tests for multiple ruby & rails versions. 7.0 was failing because messages used different kind of quote